static/app/utils/replays/mergeBreadcrumbEntries.tsx

```diff
@@ -0,0 +1,23 @@
+import {Entry, EntryType, Event} from 'sentry/types/event';
+
+/**
+ * Merge all breadcrumbs from each Event in the `events` array
+ */
+export default function mergeBreadcrumbEntries(events: Event[]): Entry {
+  const allValues = events.flatMap(event =>
+    event.entries.flatMap((entry: Entry) =>
+      entry.type === EntryType.BREADCRUMBS ? entry.data.values : []
+    )
+  );
+
+  const stringified = allValues.map(value => JSON.stringify(value));
+  const deduped = Array.from(new Set(stringified));
+  const values = deduped.map(value => JSON.parse(value));
+
+  return {
+    type: EntryType.BREADCRUMBS,
+    data: {
+      values,
+    },
+  };
+}
```

Comment:
this is copied over from [static/app/views/replays/utils/mergeBreadcrumbsEntries.tsx](https://github.com/getsentry/sentry/blob/master/static/app/views/replays/utils/mergeBreadcrumbsEntries.tsx) but without the `notRRWebTransaction` fitler.

That filter is still interesting, but in order to make things composable... we should do that filtering later, in a composable way.